sections/shared/Layout/SideNav/SubMenu.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from 'react';
+import Link from 'next/link';
+import { createPortal } from 'react-dom';
+import { useTranslation } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { FlexDivColCentered } from 'styles/common';
+import { SIDE_NAV_WIDTH } from 'constants/ui';
+
+import { linkCSS } from 'styles/common';
+import { SubMenuLink } from 'sections/shared/Layout/constants';
+
+type SubMenuConfiguration = {
+	routes: SubMenuLink[] | null;
+	topPosition: number | null;
+};
+
+type SubMenuProps = {
+	config: SubMenuConfiguration;
+	currentPath: string;
+};
+
+const SubMenu: FC<SubMenuProps> = ({ config, currentPath }) => {
+	const { t } = useTranslation();
+	return typeof document !== 'undefined'
+		? createPortal(
```

Comment:
Using Portal here as the SideNav has a `fixed` position, which means there is no way to put the subMenu below with `z-index`. The only way is to send this component to the root.

Reply:
mice!